packages/main/src/types/BreadcrumbsDesign.js

```diff
@@ -0,0 +1,42 @@
+import DataType from "@ui5/webcomponents-base/dist/types/DataType.js";
+
+/**
+ * @lends sap.ui.webcomponents.main.types.BreadcrumbsDesign.prototype
+ * @public
+ */
+const BreadcrumbsTypes = {
```

Comment:
btw here the 2 types are defined in accordance with the UX spec at:
https://ux.wdf.sap.corp/fiori-design-web/breadcrumb/#types